src/js/msp/MSPCodes.js

```diff
@@ -35,7 +35,7 @@ const MSPCodes = {
     MSP_SET_CF_SERIAL_CONFIG:       55,
     MSP_VOLTAGE_METER_CONFIG:       56,
     MSP_SET_VOLTAGE_METER_CONFIG:   57,
-    MSP_SONAR:                      58,
+    MSP_SONAR_ALTITUDE:             58, // renamed in MSP 1.46 from MSP_SONAR to same as in firmware
```

Comment:
This assumes the sonar is placed downwards. We need to prepare for the possibility of object detection and avoidance using this sensor. So I think we need to leave this as is and rename in firmware when we start improving the sonar code.

Reply:
Right, no isuse, I recover